src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -159,6 +170,26 @@ private void runGcloudCommand(List<String> args, String... topLevelCommand)
     processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private String runSynchronousGcloudCommand(List<String> args)
```

Comment:
This method could use some Javadoc explaining why it follows a separate convention to the other methods here.